webapp/channels/src/selectors/views/custom_status.ts

```diff
@@ -26,7 +26,16 @@ export function makeGetCustomStatus(): (state: GlobalState, userID?: string) =>
         (state: GlobalState, userID?: string) => (userID ? getUser(state, userID) : getCurrentUser(state)),
         (user) => {
             const userProps = user?.props || {};
-            return userProps.customStatus ? JSON.parse(userProps.customStatus) : undefined;
+            let customStatus;
+            if (userProps.customStatus) {
+                try {
+                    customStatus = JSON.parse(userProps.customStatus);
+                } catch (error) {
+                    // eslint-disable-next-line no-console
+                    console.log('Unable to parse custom status', error);
```

Comment:
I think it would be better to just do nothing here since, as this is written, it'll flood the console with logs whenever this recomputes